AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/Model/AwsCryptographyKeyStoreTypes.dfy

```diff
@@ -39,7 +39,7 @@ module {:extern "software.amazon.cryptography.keystore.internaldafny.types" } Aw
     nameonly branchKey: Secret
   )
   datatype ClobberMutationLockInput = | ClobberMutationLockInput (
```

Comment:
just making sure, for the smithy models, we are opting to go against smithy style convention, right?
https://smithy.io/2.0/guides/style-guide.html#naming